### PR TITLE
Update shared storage doc

### PR DESCRIPTION
**Pull request by J-shang**

### CODE REVIEW

File: docs/en_US/Compression/AutoCompression.rst [hunk 1]
```diff
@@ -12,6 +12,9 @@ The main differences are as follows:
 * Need to provide the model to be compressed, and the model should have already been pre-trained.
 * No need to set ``trial_command``, additional need to set ``auto_compress_module`` as ``AutoCompressionExperiment`` input.
 
+.. note::
+    Auto compression only support TPE Tuner, Random Search Tuner, Anneal Tuner, Evolution Tuner right now.
```
Comment (acured) [inline docs/en_US/Compression/AutoCompression.rst:16]: support => supports

File: docs/en_US/Tutorial/HowToUseSharedStorage.rst [hunk 2]
```diff
@@ -13,6 +13,12 @@ And we will develop more practical features in the future based on shared storag
     Shared storage is currently in the experimental stage. We suggest use AzureBlob under Ubuntu/CentOS/RHEL, and NFS under Ubuntu/CentOS/RHEL/Fedora/Debian for remote.
     And make sure your local machine can mount NFS or fuse AzureBlob and the machine used in training service has ``sudo`` permission without password. We only support shared storage under training service with reuse mode for now.
 
+.. note::
+    What is the difference between training service native storage and shared storage? Training service native storage is usually provided by the specific training service.
+    e.g., the local storage on remote machine in remote mode, the provided storage in openpai mode. These storages might not easy to use, e.g., users have to upload datasets to all remote machines to train the model.
```
Comment (acured) [inline docs/en_US/Tutorial/HowToUseSharedStorage.rst:18]: e.g. => E.g.

File: docs/en_US/Compression/AutoCompression.rst (hunk 1 above)
Comment (J-shang) [reply docs/en_US/Compression/AutoCompression.rst]: THX, fix it

File: docs/en_US/Tutorial/HowToUseSharedStorage.rst (hunk 2 above)
Comment (J-shang) [reply docs/en_US/Tutorial/HowToUseSharedStorage.rst]: done

**Review by acured — APPROVED** [review]

**Review by SparkSnail — APPROVED** [review]

**Pull request merged**